db_stress_tool/multi_ops_txns_stress.cc

```diff
@@ -1056,6 +1152,10 @@ void MultiOpsTxnsStressTest::VerifyDb(ThreadState* thread) const {
     std::string iter_ub_str(buf, sizeof(buf));
     Slice iter_ub = iter_ub_str;
 
+    EncodeFixed32(buf, Record::kPrimaryIndexId);
+    std::reverse(buf, buf + sizeof(buf));
+    std::string start_key(buf, sizeof(buf));
```

Comment:
Could be a Slice to avoid copy, and this pattern is copy-pasted all over the file.

Reply:
Let me update when applicable.

Reply:
Sometimes, the patter is as follows
```
char scratch_pad[4];
// do something on scratch_pad and create a string
// do something else on scratch_pad and create a string
```

In such cases, we actually need to create strings to store the data from the scratch pad buffer.